src/components/Logout.js

```diff
@@ -1,26 +1,26 @@
-import React from 'react';
+// import React from 'react';
```

Comment:
instead of commenting put the enter component why not leave a basic component like
```
import React from 'react';

const Logout = () => (
  <div>
    <h2>Logout Component</h2>
  </div>
);
export default Logout;
```